astropy/units/format/console.py

```diff
@@ -67,42 +67,39 @@ def format_exponential_notation(cls, val):
 
     @classmethod
     def to_string(cls, unit, inline=True):
-        if isinstance(unit, core.CompositeUnit):
-            if unit.scale == 1:
-                s = ""
+        if unit.scale == 1:
+            s = ""
+        else:
```

Comment:
In personal use I haven't noticed a case when `unit.scale` != 1. From context I'm guessing this happens for exponentials, but more specifically... when is the scale not 1?  Maybe a comment is warranted?

Reply:
You get it most often by decomposing. E.g.,
```
u.Ry.decompose()
# Unit("2.17987e-18 kg m2 / s2")
```
(the number is the scale).